subsys/mpsl/fem/mpsl_fem.c

```diff
@@ -409,7 +409,24 @@ static int fem_simple_gpio_configure(void)
 		}
 	};
 
-	err = ppi_channel_alloc(cfg.ppi_channels, ARRAY_SIZE(cfg.ppi_channels));
+	IF_ENABLED(CONFIG_HAS_HW_NRF_PPI,
+		   (err = ppi_channel_alloc(cfg.ppi_channels, ARRAY_SIZE(cfg.ppi_channels));));
+	IF_ENABLED(CONFIG_HAS_HW_NRF_DPPIC,
+		   (err = ppi_channel_alloc(cfg.dppi_channels, ARRAY_SIZE(cfg.dppi_channels));));
+	if (err) {
+		return err;
+	}
+
+	IF_ENABLED(CONFIG_SOC_SERIES_NRF53X,
```

Comment:
Any reason you are using CONFIG_SOC_SERIES_NRF53X instead of CONFIG_HAS_HW_NRF_DPPIC here?

Reply:
It was done this way for nrf21540, but I guess using CONFIG_HAS_HW_NRF_DPPIC would be more future-proof.

Reply:
I have updated the PR to fix this :slightly_smiling_face:

Reply:
Then, I would suggest to update Line.117 accordingly as well as nrf21540_gpio. @jciupis Any comment?